### DISCUSSION TITLE
Difference Solo 2A+ and 2C+

### DISCUSSION THREAD

**Discussion by sbusch** [solokeys/solo2#175] [Q&A]
I own two Solo 2A and now I often read Solo 2A+ and 2C+, e.g. on https://solokeys.com/collections/all and I wonder what the difference is?

Is it this [minor redesign](https://www.kickstarter.com/projects/conorpatrick/solo-v2-safety-net-against-phishing/posts/3561679#h%3AMinor-Redesign:~:text=picking%20%26%20placing%20components.-,Minor%20Redesign,-When%20we%20launched)?

Regards,
Sebastian

**Comment by Jackenmen** [comment]
I'm also wondering about this - the product pages for Solo 2C and Solo 2 USB-A note that these keys do not have NFC functionality *but* the keys I got (2x Solo 2 USB-A which is how solo2-cli reports them as so I'm pretty sure they're not 2A+) from Indiegogo campaign were at least *supposed to* have NFC (and afaik they at least have the needed antenna as my phone can detect it - though I was never actually able to successfully use the key over NFC).
